tools/pylib/boutdata/data.py

```diff
@@ -247,6 +248,44 @@ def __init__(self, filename, name="root"):
 
                         section[line[:eqpos].strip()] = value
 
+    def write(self, filename=None, overwrite=False):
+        """ Write to BOUT++ options file
+
+        By default this will overwrite the original input file, prompting the
+        user for confirmation.
+        Note, no comments from the original input file are transferred to the
+        new one.
+
+        Parameters
+        ----------
+        filename : str
+            Path of the file to write
+            (defaults to path of the file that was read in)
+        force_overwrite : bool
```

Comment:
Comments should be updated: user isn't prompted anymore but an exception is raised

Reply:
This and also the previous comment `force_overwrite`->`overwrite` are fixed now.